source/system/structs/VectorList.ooc

```diff
@@ -51,9 +51,11 @@ VectorList: class <T> extends List<T>{
 		this _count -= 1
 		result
 	}
-	removeAt: override func (index: Int) {
-		this _vector copy(index + 1, index)
-		this _count -= 1
+	removeAt: override func (index: Int) { this removeRange(index, index + 1) }
+	removeRange: func (start, stop: Int) {
```

Comment:
Most of the functions in this class takes `start, end: Int` but that does not matter for me (but maybe it does for @marcusnaslund :) )